src/ui/native.js

```diff
@@ -12,18 +12,21 @@ import { Device } from '@capacitor/device'
 Vue.mixin(i18nPlugin)
 Vue.mixin(capacitor)
 
-const app = (global['Floccus'] = new Vue({
-  el: '#app',
-  router,
-  store,
-  vuetify,
-  render: (h) => h(App),
-}))
-
-Device.getLanguageCode()
-  .then(({ value }) => {
-    i18n.setLocales([value])
-    i18n.load()
-  })
+const app = () => {
+  Device.getLanguageCode()
```

Comment:
Apparently we can also use navigator.langauges (same as above) here: https://github.com/ionic-team/capacitor-plugins/issues/799#issuecomment-1026989816

Reply:
Changed to use `navigator.languages` for languages.